assets/stylesheets/_components.scss

```diff
@@ -129,14 +130,14 @@
 @import 'components/seo-preview-pane/style';
 @import 'components/signup-form/style';
 @import 'components/tooltip/style';
-@import 'components/upgrades/credit-card-form/style';
 @import 'components/upgrades/credit-card-number-input/style';
 @import 'components/user/style';
 @import 'components/version/style';
 @import 'components/vertical-nav/style';
 @import 'components/vertical-nav/item/style';
 @import 'components/vertical-menu/style';
 @import 'components/web-preview/style';
+@import 'blocks/credit-card-form/style';
```

Comment:
@mtias I wanted to put this line in alphabetical order, but there were some issues with the order of CSS styles applied. If you think about it more, it really makes sense to put all CSS styles of blocks after styles of components. This would keep logical order of the pieces required to build a page: component > block > page.  What do you think about it?